src/CHANGES.md

```diff
@@ -375,7 +375,7 @@
 -   \[MISC] Typo and style [#378](https://github.com/bids-standard/bids-specification/pull/378) ([TheChymera](https://github.com/TheChymera))
 -   \[FIX] divide readme into 3 parts [#374](https://github.com/bids-standard/bids-specification/pull/374) ([sappelhoff](https://github.com/sappelhoff))
 -   \[FIX] Refer to BIDS consistently, instead of "\<Modality>-BIDS" [#366](https://github.com/bids-standard/bids-specification/pull/366) ([sappelhoff](https://github.com/sappelhoff))
--   \[FIX] Change recommended anonymization date from 1900 to 1925 [#363](https://github.com/bids-standard/bids-specification/pull/363) ([alexrockhill](https://github.com/alexrockhill))
+-   \[FIX] Change recommended deidentification date from 1900 to 1925 [#363](https://github.com/bids-standard/bids-specification/pull/363) ([alexrockhill](https://github.com/alexrockhill))
```

Comment:
We should not modify CHANGES.md. If there's a typo in a PR, we can edit the PR and it will get updated next time the changelog generator is run. In this case, I think it's okay to leave, since it accurately describes the PR.

Reply:
PR #363 is precisely about deidentififcation, not anonymisation - shifting dates so that "_chances of identifying a person based on the date and time of their scan would be decreased_". I have actually modified the paragraph modified by #363:
https://github.com/bids-standard/bids-specification/blob/01025da63297468a3801f8fe43f97a140f2f9551/src/modality-agnostic-files.md?plain=1#L537

With that said, perhaps it is not worth chasing down this occurrence of anonymisation.